src/PolyhedralGeometry/Polyhedron/properties.jl

```diff
@@ -1246,93 +1246,107 @@ function support_function(P::Polyhedron{T}; convention = :max) where T<:scalar_t
+_cmp_string(::Val{:eq}) = "="
+
 @doc raw"""
-    print_constraints(A::AnyVecOrMat, b::AbstractVector; trivial::Bool = false, numbered::Bool = false)
+    print_constraints(A::AnyVecOrMat, b::AbstractVector; trivial = false, numbered = false, io = stdout, cmp = :lte)
 
 Pretty print the constraints given by $P(A,b) = \{ x |  Ax ≤ b \}$.
 
-Trivial inequalities are counted but omitted. They are included if `trivial` is
-set to `true`.
+# Keyword Arguments
+- `trivial::Bool`: If `true`, include trivial inequalities.
+- `numbered::Bool`: If `true`, the each constraint is printed with the index corresponding to the input `AnyVecOrMat`.
+- `io::IO`: Target `IO` where the  constraints are printed to.
+- `cmp::Symbol`: Defines the string used for the comparison sign; supports `:lte` (less than or equal) and `:eq` (equal).
+
+Trivial inequalities are always counted for numbering, even when omitted.
 
 # Examples
 ```jldoctest
 julia> print_constraints([-1 0 4 5; 4 4 4 3; 1 0 0 0; 0 0 0 0; 0 0 0 0; 9 9 9 9], [0, 1, 2, 3, -4, 5]; numbered = true)
-1: -x₁ + 4*x₃ + 5*x₄ ≦ 0
-2: 4*x₁ + 4*x₂ + 4*x₃ + 3*x₄ ≦ 1
-3: x₁ ≦ 2
-5: 0 ≦ -4
-6: 9*x₁ + 9*x₂ + 9*x₃ + 9*x₄ ≦ 5
+1: -x_1 + 4*x_3 + 5*x_4 <= 0
+2: 4*x_1 + 4*x_2 + 4*x_3 + 3*x_4 <= 1
+3: x_1 <= 2
+5: 0 <= -4
+6: 9*x_1 + 9*x_2 + 9*x_3 + 9*x_4 <= 5
 
 julia> print_constraints([-1 0 4 5; 4 4 4 3; 1 0 0 0; 0 0 0 0; 0 0 0 0; 9 9 9 9], [0, 1, 2, 3, -4, 5]; trivial = true)
--x₁ + 4*x₃ + 5*x₄ ≦ 0
-4*x₁ + 4*x₂ + 4*x₃ + 3*x₄ ≦ 1
-x₁ ≦ 2
-0 ≦ 3
-0 ≦ -4
-9*x₁ + 9*x₂ + 9*x₃ + 9*x₄ ≦ 5
+-x_1 + 4*x_3 + 5*x_4 <= 0
+4*x_1 + 4*x_2 + 4*x_3 + 3*x_4 <= 1
+x_1 <= 2
+0 <= 3
+0 <= -4
+9*x_1 + 9*x_2 + 9*x_3 + 9*x_4 <= 5
 ```
 """
-function print_constraints(A::AnyVecOrMat, b::AbstractVector; trivial::Bool = false, numbered::Bool = false, io::IO = stdout, cmp::String = "≦")
-    for i in 1:length(b)
-        terms = Vector{String}(undef, size(A)[2])
-        first = true
-        for j in 1:size(A)[2]
-            if iszero(A[i, j])
-                terms[j] = ""
-            else
-                if isone(A[i, j]) || isone(-A[i, j])
-                    terms[j] = first ? string(isone(A[i, j]) ? "x" : "-x" , ['₀'+ d for d in digits(j)]...) :
-                        string(isone(A[i, j]) ? " + x" : " - x", ['₀'+ d for d in digits(j)]...)
-                else
-                    terms[j] = first ? string(_constraint_string(A[i, j]), "*x", ['₀'+ d for d in digits(j)]...) :
-                        string(A[i, j] < zero(A[i, j]) ? string(" - ", _constraint_string(-A[i, j])) : string(" + ", _constraint_string(A[i, j])), "*x", ['₀'+ d for d in digits(j)]...)
-                end
-                first = false
-            end
-        end
-        if first
-            if b[i] >= 0 && !trivial
-                continue
-            end
-            terms[1] = "0"
+function print_constraints(A::AnyVecOrMat, b::AbstractVector; trivial::Bool = false, numbered::Bool = false, io::IO = stdout, cmp::Symbol = :lte)
```

Comment:
The julia style guide states that `IO` arguments come first (see https://docs.julialang.org/en/v1/manual/style-guide/#Write-functions-with-argument-ordering-similar-to-Julia-Base). I don't know if we should follow that here